packages/now-cli/src/util/dev/server.ts

```diff
@@ -570,7 +570,8 @@ export default class DevServer {
       if (builders) {
         const { defaultRoutes, error: routesError } = await detectRoutes(
           files,
-          builders
+          builders,
+          false // TODO: enable after implementing `handle: miss`
```

Comment:
We need to tie this to a temporary now.json option

Reply:
I am not implementing `now dev` yet so I'll set this to default as `false` and avoid changing this file.

Reply:
How are we supposed to test the feature if we can't enable it?

Reply:
@dav-is There are two places `detectRoutes()` is called.

1. The `now dev` server which you commented on here
2. The `api-deployments` backend which will be modified once this PR is released

I am going to set the flag in `api-deployments` based on the now.json properties.